client/planning-extension/src/index.css

```diff
@@ -0,0 +1,14 @@
+/* ASSIGNMENTS OVERVIEW START */
+
+.assignments-overview--item {
+    display: block;
+    width: 100%;
+    padding: 0;
+    text-align: left;
+}
+
+.assignments-overview--item + .assignments-overview--item {
+    margin-top: 6px;
```

Comment:
Would `margin-top: 8px` work here, to stay in line with the UI framework's Layout Principles?